### PR TITLE
Create UI with RecyclerView and Google Map

### PR DESCRIPTION
**Pull request by xella**
https://github.com/gymnasion/CashgroupAtms/projects/1#card-5847225

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
+        android:required="true"/>
     <application
         android:name=".CashgroupatmsApp"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
+        android:theme="@style/Theme.AppCompat.Light.NoActionBar">
```
Comment (ivanfenenko) [inline app/src/main/AndroidManifest.xml]: AppTheme extends the same style

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,15 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="io.gymnasion.cashgroupatms">
-
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-feature
```
Comment (ivanfenenko) [inline app/src/main/AndroidManifest.xml]: This is unnecessary

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -1,15 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="io.gymnasion.cashgroupatms">
-
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
```
Comment (ivanfenenko) [inline app/src/main/AndroidManifest.xml]: This is unnecessary

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -1,9 +1,38 @@
...
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:map="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:orientation="vertical"
     tools:context="io.gymnasion.cashgroupatms.MainActivity">
 
-</android.support.constraint.ConstraintLayout>
+    <android.support.v7.widget.Toolbar
```
Comment (ivanfenenko) [inline app/src/main/res/layout/activity_main.xml:11]: background
theme
popupTheme
titleTextColor

These are unnecessary

**Review by ivanfenenko — CHANGES_REQUESTED** [review]
I would recommend to use Butterknife library to inject views

**Comment by xella** [comment]
@ivanfenenko I updated the pull request

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -1,9 +1,34 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:map="http://schemas.android.com/apk/res-auto"
```
Comment (ivanfenenko) [inline app/src/main/res/layout/activity_main.xml]: > xmlns:app="http://schemas.android.com/apk/res-auto"
 It's equal to this line. 


So you can use it as `app:cameraZoom="3"`

**Review by ivanfenenko — CHANGES_REQUESTED** [review]

**Review by ivanfenenko — APPROVED** [review]

**Pull request merged**